.github/workflows/pull_requests.yml

```diff
@@ -0,0 +1,11 @@
+name: Pull request
+on: pull_request
+
+jobs:
+  commitsar:
+    runs-on: ubuntu-latest
```

Comment:
Is this safe? Should we pin to an LTS version?

Reply:
@dwaxemberg I checked, and apparently that is how github does action 🤷‍♂